keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -483,28 +512,34 @@ the additional defaulting functions, automatically call them prior to
 the existing defaulting functions
 2. Update kube-openapi to parse the `+default` marker and include it in
```

Comment:
I miss the discussion and conclusion about publishing the OpenAPI defaults. For CRDs today we **do not** publish them intentionally. I suggest to do the same here.

Reply:
> For CRDs today we do not publish them intentionally.

I didn't know that. I don't really understand it, but I guess I'm fine if we do the same. As long as server-side apply is able to read them since that's one of the reason why we're building that mechanism.

Reply:
SSA will see the full schema. This happens as part of the OpenAPI v2 conversion.

Reply:
Have you added a clarification anywhere?

Reply:
No, not yet, I'm not super satisfied to hide them. it means we can't generate a good "openapi" example to test the behavior in kube-openapi without re-adding the defaults, it means client can't merge properly using listmapkey because they don't have the default either. The more expressive the schema the better, we should keep the defaults, and we should probably do so for CRDs too.

Reply:
See my text in the gdoc. I still think we have to describe our plan.